Makefile

```diff
@@ -43,8 +43,14 @@ test:
 	# cd ${BUILD_DIR}; \
 	# godep go test -v ./... 2>&1 | go2xunit -output ${TEST_REPORT} ; \
 	# cd - >/dev/null
+
+	# Compile the binary
+	cd cmd/cdi/; go install
+
+	cd ../..
 	# Just doing basic test now
-	go test ./...
+	go clean -testcache
```

Comment:
This was super strange .. after I deleted the binary my tests were passing still..30min of WTF..

Clear the cache to avoid this..